router/internal/requestlogger/subgraphlogger.go

```diff
@@ -43,19 +43,22 @@ func NewSubgraphAccessLogger(logger *zap.Logger, opts SubgraphOptions) *Subgraph
 	}
 }
 
-func (h *SubgraphAccessLogger) WriteRequestLog(respInfo *resolve.ResponseInfo, subgraphFields []zap.Field) {
+func (h *SubgraphAccessLogger) GetRequestFields(respInfo *resolve.ResponseInfo, subgraphFields []zap.Field) []zap.Field {
```

Comment:
I recommend to name this function "RequestFields":
https://chatgpt.com/share/675fdb6a-3738-8010-853f-efa3fa7beee8

Reply:
Thanks for that authoritative source 😆